dataplanes/vpp/build/Dockerfile.vpp

```diff
@@ -1,6 +1,6 @@
 FROM ubuntu:bionic
 ARG DEBIAN_FRONTEND=noninteractive
-ARG REPO=release
+ARG REPO=1807
```

Comment:
Don't fix the repo.  You still want the release repo.  Instead use"

```bash
apt-get install <package name>=<version>
```
Where version is 18.07.1

Reply:
Oh, ok, thank you

Reply:
The fdio/1807 repo tracks the packages from the stable/1807 branch... which has moved on since the last 18.07.1 release ( which is in fdio/release ) :)